superset/views/csv_import.py

```diff
@@ -352,16 +370,18 @@ def _get_existing_database(self, database_id: int, schema=None) -> models.Databa
                         database.database_name, schema
                     )
                 )
-                raise SchemaNotAllowedCsvUpload(message)
+                raise SchemaNotAllowedCsvUpload(message, None)
             return database
-        except NoResultFound:
-            raise NoResultFound("No database was found with id {}".format(database_id))
-        except MultipleResultsFound:
+        except NoResultFound as e:
+            raise NoResultFound(
+                "No database was found with id {}".format(database_id), e
+            )
+        except MultipleResultsFound as e:
             raise MultipleResultsFound(
-                "Multiple databases were found with id {}".format(database_id)
+                "Multiple databases were found with id {}".format(database_id), e
```

Comment:
Do we wanna give out IDs to the user? or use the db-name here even though that's not technically what's the problem?